sklearn/preprocessing/tests/test_discretization.py

```diff
@@ -3,8 +3,10 @@
 import numpy as np
 from six.moves import range
 import warnings
+import scipy.sparse as sp
```

Comment:
Nit: Move this above `six.moves`, so it is alphabetical.

Reply:
In fact you should use sklearn.externals.six to not have dependencies on six